README.md

```diff
@@ -71,6 +71,22 @@ This starts an S3 server on port 8000.
 The default access key is accessKey1 with
 a secret key of verySecretKey1.
 
+## Run it with environment variables
```

Comment:
I would do this as a separate section (not a run it) with a heading such as "Setting your own access key and secret key pairs"

Then I would say there are 2 options for setting your own, the env variables or the authdata.json file.